# Finnish translations for PROJECT.
# Copyright (C) 2024 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2024.
#
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2024-03-05 22:39+0200\n"
"PO-Revision-Date: 2024-02-19 22:17+0200\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: fi\n"
"Language-Team: fi <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.13.0\n"

#: api/admin/announcement_list_validator.py:35
#, python-format
msgid "Invalid announcement list format: %(announcements)r"
msgstr ""

#: api/admin/announcement_list_validator.py:50
#, python-format
msgid "Too many announcements: maximum is %(maximum)d"
msgstr ""

#: api/admin/announcement_list_validator.py:62
#, python-format
msgid "Duplicate announcement ID: %s"
msgstr ""

#: api/admin/announcement_list_validator.py:71
#, python-format
msgid "Invalid announcement format: %(announcement)r"
msgstr ""

#: api/admin/announcement_list_validator.py:85
#, python-format
msgid "Invalid announcement ID: %(id)s"
msgstr ""

#: api/admin/announcement_list_validator.py:95
#, python-format
msgid "Missing required field: %(field)s"
msgstr ""

#: api/admin/announcement_list_validator.py:141
#, python-format
msgid "Value too short (%(length)d versus %(limit)d characters): %(value)s"
msgstr ""

#: api/admin/announcement_list_validator.py:153
#, python-format
msgid "Value too long (%(length)d versus %(limit)d characters): %(value)s"
msgstr ""

#: api/admin/announcement_list_validator.py:188
#, python-format
msgid "Value for %(field)s is not a date: %(date)s"
msgstr ""

#: api/admin/announcement_list_validator.py:201
#, python-format
msgid "Value for %(field)s must be no earlier than %(minimum)s"
msgstr ""

#: api/admin/problem_details.py:9
msgid "Admin auth not configured"
msgstr ""

#: api/admin/problem_details.py:10
msgid "This circulation manager has not been configured to authenticate admins."
msgstr ""

#: api/admin/problem_details.py:16
msgid "Admin auth mechanism not configured"
msgstr ""

#: api/admin/problem_details.py:17
msgid ""
"This circulation manager has not been configured to authenticate admins "
"with the mechanism you used"
msgstr ""

#: api/admin/problem_details.py:25
msgid "Invalid admin credentials"
msgstr ""

#: api/admin/problem_details.py:26
msgid "Valid library staff credentials are required."
msgstr ""

#: api/admin/problem_details.py:32
msgid "Admin not authorized"
msgstr ""

#: api/admin/problem_details.py:33
msgid "Your admin account is not authorized to make this request."
msgstr ""

#: api/admin/problem_details.py:39
msgid "Invalid CSRF token"
msgstr ""

#: api/admin/problem_details.py:40
msgid "There was an error saving your changes."
msgstr ""

#: api/admin/problem_details.py:46
msgid "Invalid edit"
msgstr ""

#: api/admin/problem_details.py:47
msgid "There was a problem with the edited metadata."
msgstr ""

#: api/admin/problem_details.py:53
msgid "Metadata refresh pending."
msgstr ""

#: api/admin/problem_details.py:54
msgid "The Metadata Wrangler is looking for new data. Check back later."
msgstr ""

#: api/admin/problem_details.py:60 api/admin/problem_details.py:61
msgid "Metadata could not be refreshed."
msgstr ""

#: api/admin/problem_details.py:67
msgid "Genre not found."
msgstr ""

#: api/admin/problem_details.py:68
msgid "One of the submitted genres does not exist."
msgstr ""

#: api/admin/problem_details.py:74
msgid "Incompatible genre."
msgstr ""

#: api/admin/problem_details.py:75
msgid "The genre is incompatible with the fiction status of the work."
msgstr ""

#: api/admin/problem_details.py:81
msgid "Erotica is for Adults Only."
msgstr ""

#: api/admin/problem_details.py:82
msgid "The Erotica genre is incompatible with the submitted Audience."
msgstr ""

#: api/admin/problem_details.py:88
msgid "Invalid series position."
msgstr ""

#: api/admin/problem_details.py:89
msgid "The series position must be a number or blank."
msgstr ""

#: api/admin/problem_details.py:95
msgid "Invalid date format."
msgstr ""

#: api/admin/problem_details.py:96
msgid "A date must be in the format YYYY-MM-DD."
msgstr ""

#: api/admin/problem_details.py:102
msgid "Unknown language."
msgstr ""

#: api/admin/problem_details.py:103
msgid "The submitted language is not one of the known languages."
msgstr ""

#: api/admin/problem_details.py:109
msgid "Unknown location."
msgstr ""

#: api/admin/problem_details.py:110
msgid "The submitted geographic location cannot be found."
msgstr ""

#: api/admin/problem_details.py:116
msgid "Unknown role."
msgstr ""

#: api/admin/problem_details.py:117
msgid "One of the submitted roles is not one of the known roles."
msgstr ""

#: api/admin/problem_details.py:123
msgid "Unknown medium."
msgstr ""

#: api/admin/problem_details.py:124
msgid "The submitted medium is not one of the known media types."
msgstr ""

#: api/admin/problem_details.py:130
msgid "Invalid rating."
msgstr ""

#: api/admin/problem_details.py:131
msgid "The rating must be a number in the rating scale."
msgstr ""

#: api/admin/problem_details.py:137
msgid "Invalid image"
msgstr ""

#: api/admin/problem_details.py:138
msgid "The submitted image is invalid."
msgstr ""

#: api/admin/problem_details.py:144
msgid "Library short name already in use"
msgstr ""

#: api/admin/problem_details.py:145
msgid ""
"The library short name must be unique, and there's already a library with"
" the specified short name."
msgstr ""

#: api/admin/problem_details.py:153
msgid "Missing collection."
msgstr ""

#: api/admin/problem_details.py:154
msgid "The specified collection does not exist."
msgstr ""

#: api/admin/problem_details.py:160
msgid "Missing analytics name."
msgstr ""

#: api/admin/problem_details.py:161
msgid "You must identify the analytics service by its name."
msgstr ""

#: api/admin/problem_details.py:167
msgid "Cannot delete collection with children"
msgstr ""

#: api/admin/problem_details.py:168
msgid ""
"The collection is the parent of at least one other collection, so it "
"can't be deleted."
msgstr ""

#: api/admin/problem_details.py:176
msgid "No protocol for new service"
msgstr ""

#: api/admin/problem_details.py:177
msgid ""
"The specified service doesn't exist. You can create it, but you must "
"specify a protocol."
msgstr ""

#: api/admin/problem_details.py:185
msgid "Unknown protocol"
msgstr ""

#: api/admin/problem_details.py:186
msgid "The protocol is not one of the known protocols."
msgstr ""

#: api/admin/problem_details.py:192
msgid "Cannot change protocol"
msgstr ""

#: api/admin/problem_details.py:193
msgid "A protocol can't be changed once it has been set."
msgstr ""

#: api/admin/problem_details.py:199
msgid "Missing service name."
msgstr ""

#: api/admin/problem_details.py:200
msgid "You must identify the service by its name."
msgstr ""

#: api/admin/problem_details.py:206
msgid "Protocol does not support parents"
msgstr ""

#: api/admin/problem_details.py:207
msgid "You attempted to add a parent but the protocol does not support parents."
msgstr ""

#: api/admin/problem_details.py:215
msgid "Protocol does not support settings"
msgstr ""

#: api/admin/problem_details.py:216
msgid ""
"You attempted to update settings for a protocol that does not support "
"settings."
msgstr ""

#: api/admin/problem_details.py:224
msgid "Missing parent"
msgstr ""

#: api/admin/problem_details.py:225
msgid "You attempted to add a parent that does not exist."
msgstr ""

#: api/admin/problem_details.py:231
msgid "No such library"
msgstr ""

#: api/admin/problem_details.py:232
msgid "A library in your request does not exist."
msgstr ""

#: api/admin/problem_details.py:238
msgid "Incomplete configuration"
msgstr ""

#: api/admin/problem_details.py:239
msgid "The configuration is missing a required field."
msgstr ""

#: api/admin/problem_details.py:245
msgid "Duplicate integration"
msgstr ""

#: api/admin/problem_details.py:246
msgid "A given site can only support one integration of this type."
msgstr ""

#: api/admin/problem_details.py:252
msgid "Integration name already in use"
msgstr ""

#: api/admin/problem_details.py:253
msgid ""
"The integration name must be unique, and there's already an integration "
"with the specified name."
msgstr ""

#: api/admin/problem_details.py:261
msgid "Integration URL already in use"
msgstr ""

#: api/admin/problem_details.py:262
msgid "There's already an integration of this type for this URL."
msgstr ""

#: api/admin/problem_details.py:268
msgid "Incompatible use of integration"
msgstr ""

#: api/admin/problem_details.py:269
msgid ""
"You tried to use an integration in a way incompatible with the goal of "
"that integration"
msgstr ""

#: api/admin/problem_details.py:277
msgid "Missing integration"
msgstr ""

#: api/admin/problem_details.py:278
msgid "The specified integration does not exist."
msgstr ""

#: api/admin/problem_details.py:284
msgid "Missing pgcrypto database extension"
msgstr ""

#: api/admin/problem_details.py:285
msgid ""
"You tried to store a password for an individual admin, but the database "
"does not have the pgcrypto extension installed."
msgstr ""

#: api/admin/problem_details.py:293
msgid "Missing admin"
msgstr ""

#: api/admin/problem_details.py:294
msgid "The specified admin does not exist."
msgstr ""

#: api/admin/problem_details.py:300
msgid "Missing service"
msgstr ""

#: api/admin/problem_details.py:301
msgid "The specified service does not exist."
msgstr ""

#: api/admin/problem_details.py:307
msgid "Invalid configuration option"
msgstr ""

#: api/admin/problem_details.py:308
msgid "The configuration has an invalid value."
msgstr ""

#: api/admin/problem_details.py:314
msgid "Invalid email format"
msgstr ""

#: api/admin/problem_details.py:315
msgid "You must enter a valid email address."
msgstr ""

#: api/admin/problem_details.py:321
msgid "Invalid URL format"
msgstr ""

#: api/admin/problem_details.py:322
msgid "You must enter a valid URL."
msgstr ""

#: api/admin/problem_details.py:328
msgid "Invalid number format"
msgstr ""

#: api/admin/problem_details.py:329
msgid "You must enter a number."
msgstr ""

#: api/admin/problem_details.py:335
msgid "Invalid external type regular expression"
msgstr ""

#: api/admin/problem_details.py:336
msgid "The specified external type regular expression does not compile."
msgstr ""

#: api/admin/problem_details.py:342
msgid "Invalid library identifier restriction regular expression"
msgstr ""

#: api/admin/problem_details.py:343
msgid ""
"The specified library identifier restriction regular expression does not "
"compile."
msgstr ""

#: api/admin/problem_details.py:351
msgid "Multiple basic authentication services"
msgstr ""

#: api/admin/problem_details.py:352
msgid ""
"Each library can only have one patron authentication service using basic "
"auth."
msgstr ""

#: api/admin/problem_details.py:360
msgid "No such patron"
msgstr ""

#: api/admin/problem_details.py:361
msgid ""
"The specified patron doesn't exist, or is associated with a different "
"library."
msgstr ""

#: api/admin/problem_details.py:369
msgid "Missing sitewide setting key"
msgstr ""

#: api/admin/problem_details.py:370
msgid "A key is required to change a sitewide setting."
msgstr ""

#: api/admin/problem_details.py:376
msgid "Missing sitewide setting value"
msgstr ""

#: api/admin/problem_details.py:377
msgid "A value is required to change a sitewide setting."
msgstr ""

#: api/admin/problem_details.py:383
msgid "Multiple sitewide services"
msgstr ""

#: api/admin/problem_details.py:384
msgid ""
"You tried to create a new sitewide service, but a sitewide service of the"
" same type is already configured."
msgstr ""

#: api/admin/problem_details.py:392
msgid "Multiple services for library"
msgstr ""

#: api/admin/problem_details.py:393
msgid "Each library can only have one service of this type."
msgstr ""

#: api/admin/problem_details.py:399
msgid "Missing custom list"
msgstr ""

#: api/admin/problem_details.py:400
msgid "The specified custom list doesn't exist."
msgstr ""

#: api/admin/problem_details.py:406
msgid "Cannot change library for custom list"
msgstr ""

#: api/admin/problem_details.py:407
msgid "A custom list's associated library cannot be changed once it is set.."
msgstr ""

#: api/admin/problem_details.py:413
msgid "Custom list name already in use"
msgstr ""

#: api/admin/problem_details.py:414
msgid "The library already has a custom list with that name."
msgstr ""

#: api/admin/problem_details.py:420
msgid "An auto update custom list cannot have entries"
msgstr ""

#: api/admin/problem_details.py:421
msgid "Entries are automatically managed for auto update custom lists"
msgstr ""

#: api/admin/problem_details.py:427
msgid "Collection not associated with library"
msgstr ""

#: api/admin/problem_details.py:428
msgid ""
"You can't add a collection to a list unless it is associated with the "
"list's library."
msgstr ""

#: api/admin/problem_details.py:436
msgid "Missing lane"
msgstr ""

#: api/admin/problem_details.py:437
msgid ""
"The specified lane doesn't exist, or is associated with a different "
"library."
msgstr ""

#: api/admin/problem_details.py:445
msgid "Cannot edit default lane"
msgstr ""

#: api/admin/problem_details.py:446
msgid "You can't change one of the default auto-generated lanes."
msgstr ""

#: api/admin/problem_details.py:452
msgid "No display name for lane"
msgstr ""

#: api/admin/problem_details.py:453
msgid "A custom lane must have a name."
msgstr ""

#: api/admin/problem_details.py:459
msgid "No custom lists for lane"
msgstr ""

#: api/admin/problem_details.py:460
msgid "A custom lane must have at least one associated list."
msgstr ""

#: api/admin/problem_details.py:466
msgid "Lane with parent and display name already exists"
msgstr ""

#: api/admin/problem_details.py:467
msgid ""
"You cannot create a lane with the same parent and display name as an "
"existing lane."
msgstr ""

#: api/admin/problem_details.py:475
msgid "Cannot show lane with hidden parent"
msgstr ""

#: api/admin/problem_details.py:476
msgid "You can only make a lane visible if its parent is already visible."
msgstr ""

#: api/admin/problem_details.py:482
msgid "The collection does not support registration"
msgstr ""

#: api/admin/problem_details.py:483
msgid "The collection does not support registration."
msgstr ""

#: api/admin/problem_details.py:489 api/admin/problem_details.py:490
msgid "Failed to run self tests."
msgstr ""

#: api/admin/problem_details.py:496
msgid "Missing identifier"
msgstr ""

#: api/admin/problem_details.py:497
msgid "No identifier was used."
msgstr ""

#: api/admin/problem_details.py:503
msgid "Source collection missing"
msgstr ""

#: api/admin/problem_details.py:504
msgid "A source collection is not present in the library."
msgstr ""

#: api/admin/problem_details.py:510
msgid "Entry not valid for library"
msgstr ""

#: api/admin/problem_details.py:511
msgid ""
"An entry in the customlist was not valid for the library being shared "
"with."
msgstr ""

#: api/admin/problem_details.py:519
msgid "Cannot delete the Custom List share"
msgstr ""

#: api/admin/problem_details.py:520
msgid "Unable to delete the shared status of the given list"
msgstr ""

#: api/admin/validator.py:61
#, python-format
msgid "\"%(email)s\" is not a valid email address."
msgstr ""

#: api/admin/validator.py:86
#, python-format
msgid "\"%(url)s\" is not a valid URL."
msgstr ""

#: api/admin/validator.py:118
#, python-format
msgid "\"%(number)s\" is not a number."
msgstr ""

#: api/admin/validator.py:125
#, python-format
msgid "%(field)s must be greater than %(min)s."
msgstr ""

#: api/admin/validator.py:133
#, python-format
msgid "%(field)s cannot be greater than %(max)s."
msgstr ""
# English translations for PROJECT.
# Copyright (C) 2024 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2024.
# 
# Translators:
# Marika Kaskinen, 2024
# 
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2024-02-19 22:26+0200\n"
"PO-Revision-Date: 2024-03-01 11:36+0000\n"
"Last-Translator: Marika Kaskinen, 2024\n"
"Language-Team: Finnish (https://app.transifex.com/national-library-of-finland/teams/183401/fi/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.13.0\n"
"Language: fi\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: api/authenticator.py:324
msgid ""
"SAML providers are configured, but secret for signing bearer tokens is not."
msgstr ""
"SAML-palveluntarjoajat on määritetty, mutta salauskoodia bearer tokenien "
"allekirjoittamista varten ei ole."

#: api/authenticator.py:563
msgid "No SAML providers are configured."
msgstr "SAML-palveluntarjoajia ei ole määritetty."

#: api/authenticator.py:572
#, python-format
msgid " The known providers are: %s"
msgstr " Tunnetut palveluntarjoajat ovat seuraavat: %s"

#: api/authenticator.py:756
msgid "Library"
msgstr "Kirjasto"

#: api/axis.py:99
msgid "Username"
msgstr "Käyttäjänimi"

#: api/axis.py:102 api/novelist.py:51
msgid "Password"
msgstr "Salasana"

#: api/axis.py:106 api/bibliotheca.py:101 api/enki.py:81 api/overdrive.py:134
#: api/overdrive.py:195
msgid "Library ID"
msgstr "Kirjaston tunnus"

#: api/axis.py:113
msgid "Server"
msgstr "Palvelin"

#: api/axis.py:120
msgid "Verify SSL Certificate"
msgstr "Vahvista SSL-varmenne"

#: api/axis.py:121
msgid ""
"This should always be True in production, it may need to be set to False to "
"use theAxis 360 QA Environment."
msgstr ""
"Tämän tulisi aina olla Tosi tuotannossa, sen arvoksi on ehkä asetettava "
"Epätosi, jotta voidaan käyttää Axis 360 QA -ympäristöä."

#: api/axis.py:127
msgid "True"
msgstr "Tosi"

#: api/axis.py:128
msgid "False"
msgstr "Epätosi"

#: api/bibliotheca.py:89
msgid "Account ID"
msgstr "Tilin tunnus"

#: api/bibliotheca.py:95
msgid "Account Key"
msgstr "Tilin avain"

#: api/bibliotheca.py:110 api/enki.py:85
msgid "Show/Hide Titles with No Available Loans"
msgstr "Näytä/piilota teokset, joita ei ole saatavilla lainattavaksi"

#: api/bibliotheca.py:112 api/enki.py:87
msgid ""
"Titles with no available loans will not be displayed in the Catalog view."
msgstr ""
"Teokset, joita ei ole saatavilla lainattavaksi, eivät näy luettelonäkymässä."

#: api/circulation.py:507
msgid "Ebook Loan Duration (in Days)"
msgstr "E-kirjan laina-aika (päivinä)"

#: api/circulation.py:509
msgid ""
"When a patron uses SimplyE to borrow an ebook from this collection, SimplyE "
"will ask for a loan that lasts this number of days. This must be equal to or"
" less than the maximum loan duration negotiated with the distributor."
msgstr ""
"Kun asiakas käyttää SimplyE-sovellusta lainatakseen e-kirjan tästä "
"kokoelmasta, SimplyE pyytää lainaa, jonka pituus on tämä määrä päiviä. Tämän"
" on oltava yhtä suuri tai pienempi kuin jakelijan kanssa neuvoteltu lainan "
"enimmäiskesto."

#: api/circulation.py:522
msgid "Default Loan Period (in Days)"
msgstr "Oletuslaina-aika (päivinä)"

#: api/circulation.py:524
msgid ""
"Until it hears otherwise from the distributor, this server will assume that "
"any given loan for this library from this collection will last this number "
"of days. This number is usually a negotiated value between the library and "
"the distributor. This only affects estimates&mdash;it cannot affect the "
"actual length of loans."
msgstr ""
"Kunnes palvelin kuulee toisin jakelijalta, se olettaa, että tämän kirjaston "
"laina tästä kokoelmasta kestää tämän määrän päiviä. Tämä luku on yleensä "
"kirjaston ja jakelijan neuvottelema arvo. Tämä vaikuttaa vain arvioihin – se"
" ei voi vaikuttaa lainojen todelliseen kestoon."

#: api/circulation.py:554
#, python-format
msgid ""
"Could not map delivery mechanism %(mechanism_name)s to internal delivery "
"mechanism!"
msgstr ""
"Toimitusmekanismia %(mechanism_name)s ei voitu määrittää sisäiseksi "
"toimitusmekanismiksi!"

#: api/circulation.py:1066
msgid "You cannot renew a loan if other patrons have the work on hold."
msgstr "Et voi uusia lainaa, jos muut asiakkaat jonottavat teosta."

#: api/circulation.py:1338
msgid "Cannot find your active loan for this work."
msgstr "Aktiivista lainaasi tälle teokselle ei löydy."

#: api/circulation.py:1345
#, python-format
msgid ""
"You already fulfilled this loan as %(loan_delivery_mechanism)s, you can't "
"also do it as %(requested_delivery_mechanism)s"
msgstr ""
"Olet jo toteuttanut tämän lainan toimitusmekanismin "
"%(loan_delivery_mechanism)s kautta, et voi toteuttaa sitä myös "
"toimitusmekanismin %(requested_delivery_mechanism)s kautta"

#: api/circulation_exceptions.py:41
#, python-format
msgid "Integration error communicating with %(service_name)s"
msgstr "Integrointivirhe kommunikoinnissa palvelun %(service_name)s kanssa"

#: api/circulation_manager.py:179
msgid "Lane does not exist"
msgstr "Polkua ei ole olemassa"

#: api/config.py:81
msgid "Internal signing secret for OAuth and SAML bearer tokens"
msgstr ""
"OAuthin ja SAML:n bearer tokenien sisäisen allekirjoittamisen salauskoodi"

#: api/config.py:86
msgid "Internal secret key for admin interface cookies"
msgstr "Hallintakäyttöliittymän evästeiden sisäinen salausavain"

#: api/config.py:91
msgid "Hostnames for web application access"
msgstr "Isäntänimet verkkosovellusten käyttöä varten"

#: api/config.py:93
msgid ""
"Only web applications from these hosts can access this circulation manager. "
"This can be a single hostname (http://catalog.library.org) or a pipe-"
"separated list of hostnames "
"(http://catalog.library.org|https://beta.library.org). You must include the "
"scheme part of the URI (http:// or https://). You can also set this to '*' "
"to allow access from any host, but you must not do this in a production "
"environment -- only during development."
msgstr ""
"Vain näiden isäntien verkkosovellukset voivat käyttää tätä lainauksen "
"hallintaa. Tämä voi olla yksittäisen isännän nimi "
"(http://catalog.library.org) tai pystyviivalla erotettu luettelo isäntien "
"nimistä (http://catalog.library.org|https://beta.library.org). Sinun on "
"sisällytettävä URIn skeema (http:// tai https://). Voit myös asettaa tämän "
"arvoksi '*', jolloin pääsy sallitaan miltä tahansa isäntäkoneelta, mutta et "
"voi tehdä tätä tuotantoympäristössä -- vain kehityksen aikana."

#: api/config.py:99
msgid "Cache time for static images and JS and CSS files (in seconds)"
msgstr ""
"Staattisten kuvien sekä JS- ja CSS-tiedostojen välimuistiaika (sekunneissa)"

#: api/config.py:107
msgid "Cache time for authentication documents (in seconds)"
msgstr "Todennusdokumenttien välimuistiaika (sekunneissa)"

#: api/config.py:114
msgid "Custom Terms of Service link"
msgstr "Mukautettujen käyttöehtojen linkki"

#: api/config.py:117
msgid ""
"If your inclusion in the SimplyE mobile app is governed by terms other than "
"the default, put the URL to those terms in this link so that librarians will"
" have access to them. This URL will be used for all libraries on this "
"circulation manager."
msgstr ""
"Jos osallistumistasi SimplyE-mobiilisovellukseen koskevat muut kuin "
"oletusehdot, lisää näiden ehtojen URL tähän linkkiin, jotta "
"kirjastonhoitajat voivat käyttää niitä. Tätä URL-osoitetta käytetään "
"kaikissa kirjastoissa tässä lainauksen hallinnassa."

#: api/config.py:123
msgid "Custom Terms of Service link text"
msgstr "Mukautettujen käyttöehtojen linkin teksti"

#: api/config.py:126
msgid ""
"Custom text for the Terms of Service link in the footer of these "
"administrative interface pages. This is primarily useful if you're not "
"connecting this circulation manager to the SimplyE mobile app. This text "
"will be used for all libraries on this circulation manager."
msgstr ""
"Käyttöehtojen linkin mukautettu teksti näiden hallinnollisten "
"käyttöliittymäsivujen alatunnisteessa. Tämä on hyödyllistä ensisijaisesti "
"silloin, jos et yhdistä tätä lainauksen hallintaa SimplyE-"
"mobiilisovellukseen. Tätä tekstiä käytetään kaikissa kirjastoissa tässä "
"lainauksen hallinnassa."

#: api/config.py:135
msgid "Scheduled announcements"
msgstr "Ajastetut ilmoitukset"

#: api/config.py:136
msgid "Announcements will be displayed to authenticated patrons."
msgstr "Ilmoitukset näytetään todennetuille asiakkaille."

#: api/custom_index.py:87
msgid "I'm Under 13"
msgstr "Olen alle 13-vuotias"

#: api/custom_index.py:88
msgid "Read children's books"
msgstr "Lue lasten kirjoja"

#: api/custom_index.py:89
msgid "I'm 13 or Older"
msgstr "Olen 13-vuotias tai vanhempi"

#: api/custom_index.py:90
msgid "See the full collection"
msgstr "Näytä koko kokoelma"

#: api/custom_index.py:98 api/custom_patron_catalog.py:171
msgid "ID of lane for patrons who are 13 or older"
msgstr "13-vuotiaiden ja sitä vanhempien asiakkaiden polun tunnus"

#: api/custom_index.py:102 api/custom_patron_catalog.py:175
msgid "ID of lane for patrons who are under 13"
msgstr "Alle 13-vuotiaiden asiakkaiden polun tunnus"

#: api/custom_patron_catalog.py:122
msgid "Send patrons to the lane with this ID."
msgstr "Lähetä asiakkaat polulle tällä tunnuksella."

#: api/ekirjasto_authentication.py:63
msgid "E-kirjasto API environment"
msgstr "E-kirjaston API-ympäristö"

#: api/ekirjasto_authentication.py:64
msgid "Select what environment of E-kirjasto accounts should be used."
msgstr "Valitse, mitä e-kirjaston ympäristöä tilien tulisi käyttää."

#: api/ekirjasto_authentication.py:81
msgid "Delegate token expire time in seconds"
msgstr "Valtuutustunnuksen vanhenemisaika sekunteina"

#: api/ekirjasto_authentication.py:82
msgid ""
"Expire time for a delegate token to authorize in behalf of a ekirjasto "
"token. This should be less than the expire time for ekirjasto token, so it "
"can be refreshed."
msgstr ""
"Määräaika, jonka kuluessa valtuutustunnuksen on annettava valtuutus "
"e-kirjaston tunnuksen puolesta. Tämän on oltava lyhyempi kuin e-kirjaston "
"tunnuksen vanhenemisaika, jotta se voidaan päivittää."

#: api/enki.py:74 api/firstbook2.py:27 api/kansas_patron.py:20
msgid "URL"
msgstr "URL-osoite"

#: api/enki.py:105
msgid "Integrate an Enki collection."
msgstr "Integroi Enki-kokoelma."

#: api/firstbook2.py:28
msgid "The URL for the First Book authentication service."
msgstr "First Bookin todennuspalvelun URL-osoite."

#: api/firstbook2.py:35
msgid "Key"
msgstr "Avain"

#: api/firstbook2.py:36
msgid "The key for the First Book authentication service."
msgstr "First Bookin todennuspalvelun avain."

#: api/millenium_patron.py:177
msgid "III Millenium Patron API"
msgstr "III Millenium -asiakkaan API"

#: api/novelist.py:42
msgid "Novelist API"
msgstr "Novelist API"

#: api/novelist.py:50
msgid "Profile"
msgstr "Profiili"

#: api/nyt.py:63
msgid "NYT Best Seller API"
msgstr "NYT Best Seller API"

#: api/nyt.py:67
msgid "API key"
msgstr "API-avain"

#: api/odl.py:85
msgid "ODL feed URL"
msgstr "ODL-syötteen URL-osoite"

#: api/odl.py:92
msgid "Library's API username"
msgstr "Kirjaston APIn käyttäjänimi"

#: api/odl.py:100
msgid "Library's API password"
msgstr "Kirjaston APIn salasana"

#: api/odl.py:108
msgid "Default Reservation Period (in Days)"
msgstr "Oletusvarausaika (päivinä)"

#: api/odl.py:109
msgid ""
"The number of days a patron has to check out a book after a hold becomes "
"available."
msgstr ""
"Asiakkaalla on näin monta päivää aikaa lainata kirja sen jälkeen, kun varaus"
" on tullut saataville."

#: api/odl.py:120
msgid "Passphrase hint"
msgstr "Salalausevihje"

#: api/odl.py:121
msgid "Hint displayed to the user when opening an LCP protected publication."
msgstr ""
"Vihje, joka näytetään käyttäjälle, kun hän avaa LCP-suojatun julkaisun."

#: api/odl.py:132
msgid "Passphrase hint URL"
msgstr "Salalausevihjeen URL-osoite"

#: api/odl.py:133
msgid ""
"Hint URL available to the user when opening an LCP protected publication."
msgstr ""
"Vihjeen URL-osoite, joka on käyttäjän saatavilla, kun hän avaa LCP-suojatun "
"julkaisun."

#: api/odl.py:144
msgid "Passphrase encryption algorithm"
msgstr "Salalauseen salausalgoritmi"

#: api/odl.py:145
msgid "Algorithm used for encrypting the passphrase."
msgstr "Salalauseen salaukseen käytettävä algoritmi."

#: api/odl2.py:46
msgid "Skipped license formats"
msgstr "Ohitetut lisenssimuodot"

#: api/odl2.py:47
msgid ""
"List of license formats that will NOT be imported into Circulation Manager."
msgstr "Luettelo lisenssimuodoista, joita EI tuoda lainauksen hallintaan."

#: api/odl2.py:59
msgid "Loan limit per patron"
msgstr "Asiakaskohtainen lainaraja"

#: api/odl2.py:60
msgid ""
"The maximum number of books a patron can have loaned out at any given time."
msgstr ""
"Asiakkaalla voi olla enintään näin monta kirjaa lainattuna kerrallaan."

#: api/odl2.py:72
msgid "Hold limit per patron"
msgstr "Asiakaskohtainen varausraja"

#: api/odl2.py:73
msgid ""
"The maximum number of books a patron can have on hold at any given time."
msgstr "Asiakkaalla voi olla enintään näin monta kirjaa varattuna kerrallaan."

#: api/opds_for_distributors.py:52
msgid "Library's username or access key"
msgstr "Kirjaston käyttäjänimi tai käyttöoikeusavain"

#: api/opds_for_distributors.py:59
msgid "Library's password or secret key"
msgstr "Kirjaston salasana tai salausavain"

#: api/overdrive.py:142
msgid "Website ID"
msgstr "Sivuston tunnus"

#: api/overdrive.py:150
msgid "Client Key"
msgstr "Asiakkaan avain"

#: api/overdrive.py:158
msgid "Client Secret"
msgstr "Asiakkaan salasana"

#: api/overdrive.py:168
msgid "Server family"
msgstr "Palvelinperhe"

#: api/overdrive.py:174
msgid "Testing"
msgstr "Testaus"

#: api/overdrive.py:184
msgid "ILS Name"
msgstr "ILS-nimi"

#: api/overdrive.py:185
msgid ""
"When multiple libraries share an Overdrive account, Overdrive uses a setting"
" called 'ILS Name' to determine which ILS to check when validating a given "
"patron."
msgstr ""
"Kun useat kirjastot käyttävät samaa Overdrive-tiliä, Overdrive määrittää "
"\"ILS-nimi\"-asetuksen perusteella mikä ILS tarkistetaan, kun asiakas "
"vahvistetaan."

#: api/problem_details.py:9
msgid "Third-party service failed."
msgstr "Kolmannen osapuolen palvelu epäonnistui."

#: api/problem_details.py:10
msgid ""
"The library could not complete your request because a third-party service "
"has failed."
msgstr ""
"Kirjasto ei pystynyt suorittamaan pyyntöäsi, koska kolmannen osapuolen "
"palvelu epäonnistui."

#: api/problem_details.py:18
msgid "Feed should be been pre-cached."
msgstr "Syötteen olisi pitänyt olla valmiiksi välimuistissa."

#: api/problem_details.py:19
msgid ""
"This feed should have been pre-cached. It's too expensive to generate "
"dynamically."
msgstr ""
"Tämän syötteen olisi pitänyt olla valmiiksi välimuistissa. Dynaaminen "
"luominen on liian kallista."

#: api/problem_details.py:27
msgid "Invalid credentials"
msgstr "Virheelliset kirjautumistiedot"

#: api/problem_details.py:28
msgid "A valid library card barcode number and PIN are required."
msgstr "Kelvollinen kirjastokortin viivakoodinumero ja PIN vaaditaan."

#: api/problem_details.py:34
msgid "Expired credentials."
msgstr "Vanhentuneet kirjautumistiedot."

#: api/problem_details.py:35
msgid "Your library card has expired. You need to renew it."
msgstr "Kirjastokorttisi on vanhentunut. Sinun on uusittava se."

#: api/problem_details.py:41
msgid "Suspended credentials."
msgstr "Jäädytetyt kirjautumistiedot."

#: api/problem_details.py:42
msgid "Your library card has been suspended. Contact your branch library."
msgstr "Kirjastokorttisi on jäädytetty. Ota yhteyttä sivukirjastoosi."

#: api/problem_details.py:48
msgid "No licenses."
msgstr "Ei lisenssejä."

#: api/problem_details.py:49
msgid "The library currently has no licenses for this book."
msgstr "Kirjastolla ei ole tällä hetkellä lisenssejä tähän kirjaan."

#: api/problem_details.py:55
msgid "No available license."
msgstr "Ei saatavilla olevaa lisenssiä."

#: api/problem_details.py:56
msgid "All licenses for this book are loaned out."
msgstr "Kaikki tämän kirjan lisenssit ovat lainassa."

#: api/problem_details.py:62
msgid "No acceptable format."
msgstr "Ei hyväksyttävää muotoa."

#: api/problem_details.py:63
msgid "Could not deliver this book in an acceptable format."
msgstr "Tätä kirjaa ei voitu toimittaa hyväksyttävässä muodossa."

#: api/problem_details.py:69
msgid "Already checked out"
msgstr "On jo lainattu"

#: api/problem_details.py:70
msgid "You have already checked out this book."
msgstr "Olet jo lainannut tämän kirjan."

#: api/problem_details.py:73
msgid ""
"You have reached your loan limit. You cannot borrow anything further until "
"you return something."
msgstr ""
"Olet saavuttanut lainarajasi. Et voi lainata enää mitään ennen kuin palautat"
" jotain."

#: api/problem_details.py:76
#, python-format
msgid ""
"You have reached your loan limit of %(limit)d. You cannot borrow anything "
"further until you return something."
msgstr ""
"Olet saavuttanut lainarajasi, joka on %(limit)d. Et voi lainata enää mitään "
"ennen kuin palautat jotain."

#: api/problem_details.py:82
msgid "Loan limit reached."
msgstr "Lainaraja saavutettu."

#: api/problem_details.py:86
msgid ""
"You have reached your hold limit. You cannot place another item on hold "
"until you borrow something or remove a hold."
msgstr ""
"Olet saavuttanut varausrajasi. Et voi varata toista teosta ennen kuin "
"lainaat jotakin tai poistat varauksen."

#: api/problem_details.py:89
#, python-format
msgid ""
"You have reached your hold limit of %(limit)d. You cannot place another item"
" on hold until you borrow something or remove a hold."
msgstr ""
"Olet saavuttanut varausrajasi, joka on %(limit)d. Et voi varata toista "
"teosta ennen kuin lainaat jotakin tai poistat varauksen."

#: api/problem_details.py:95
msgid "Limit reached."
msgstr "Raja saavutettu."

#: api/problem_details.py:102
msgid "Outstanding fines."
msgstr "Maksamattomia sakkoja."

#: api/problem_details.py:103
msgid "You must pay your outstanding fines before you can borrow more books."
msgstr ""
"Sinun on maksettava maksamattomat sakkosi ennen kuin voit lainata lisää "
"kirjoja."

#: api/problem_details.py:109
msgid "Could not issue loan."
msgstr "Teosta ei voi lainata."

#: api/problem_details.py:110
msgid "Could not issue loan (reason unknown)."
msgstr "Teosta ei voi lainata (syy tuntematon)."

#: api/problem_details.py:116
msgid "Could not place hold."
msgstr "Varausta ei voi tehdä."

#: api/problem_details.py:117
msgid "Could not place hold (reason unknown)."
msgstr "Varausta ei voi tehdä (syy tuntematon)."

#: api/problem_details.py:123
msgid "Could not renew loan."
msgstr "Lainaa ei voitu uusia."

#: api/problem_details.py:124
msgid "Could not renew loan (reason unknown)."
msgstr "Lainaa ei voitu uusia (syy tuntematon)."

#: api/problem_details.py:130
msgid "No longer in collection."
msgstr "Ei ole enää kokoelmassa."

#: api/problem_details.py:131
msgid "This book was recently removed from the collection."
msgstr "Tämä kirja poistettiin äskettäin kokoelmasta."

#: api/problem_details.py:137
msgid "No active loan."
msgstr "Ei aktiivista lainaa."

#: api/problem_details.py:138
msgid "You can't do this without first borrowing this book."
msgstr "Et voi tehdä tätä ilman, että lainaat ensin tämän kirjan."

#: api/problem_details.py:144
msgid "No active hold."
msgstr "Ei aktiivista varausta."

#: api/problem_details.py:145
msgid "You can't do this without first putting this book on hold."
msgstr "Et voi tehdä tätä ilman, että varaat ensin tämän kirjan."

#: api/problem_details.py:151
msgid "No active loan or hold."
msgstr "Ei aktiivista lainaa tai varausta."

#: api/problem_details.py:152
msgid ""
"You can't do this without first borrowing this book or putting it on hold."
msgstr "Et voi tehdä tätä ilman, että ensin lainaat tai varaat tämän kirjan."

#: api/problem_details.py:158
msgid "Loan not found."
msgstr "Lainaa ei löytynyt."

#: api/problem_details.py:159
msgid "You don't have a loan with the provided id."
msgstr "Sinulla ei ole lainaa annetulla tunnuksella."

#: api/problem_details.py:165
msgid "Hold not found."
msgstr "Varausta ei löytynyt."

#: api/problem_details.py:166
msgid "You don't have a hold with the provided id."
msgstr "Sinulla ei ole varausta annetulla tunnuksella."

#: api/problem_details.py:172
msgid "Could not mirror local state to remote."
msgstr "Paikallista tilaa ei voitu peilata etätilaan."

#: api/problem_details.py:173
msgid ""
"Could not convince a third party to accept the change you made. It's likely "
"to show up again soon."
msgstr ""
"Kolmatta osapuolta ei saatu hyväksymään tekemääsi muutosta. Se näkyy "
"todennäköisesti pian uudelleen."

#: api/problem_details.py:181
msgid "No such lane."
msgstr "Tällaista polkua ei ole."

#: api/problem_details.py:182
msgid "You asked for a nonexistent lane."
msgstr "Pyysit polkua, jota ei ole."

#: api/problem_details.py:188
msgid "No such list."
msgstr "Tällaista listaa ei ole."

#: api/problem_details.py:189
msgid "You asked for a nonexistent list."
msgstr "Pyysit listaa, jota ei ole."

#: api/problem_details.py:195
msgid "Cannot delete list."
msgstr "Listaa ei voi poistaa."

#: api/problem_details.py:196
msgid "It is not possible to delete a shared list."
msgstr "Jaettua listaa ei voi poistaa."

#: api/problem_details.py:202
msgid "No such collection."
msgstr "Tällaista kokoelmaa ei ole."

#: api/problem_details.py:203
msgid "You asked for a nonexistent collection."
msgstr "Pyysit kokoelmaa, jota ei ole."

#: api/problem_details.py:209
msgid "Forbidden by policy."
msgstr "Käytännöt kieltävät."

#: api/problem_details.py:210
msgid "Library policy prevents us from carrying out your request."
msgstr "Kirjaston käytännöt estävät meitä toteuttamasta pyyntöäsi."

#: api/problem_details.py:214
msgid ""
"Library policy considers this title inappropriate for your patron type."
msgstr ""
"Kirjaston käytäntöjen mukaan tämä teos on sopimaton asiakastyypillesi."

#: api/problem_details.py:221 api/problem_details.py:222
msgid "Could not fulfill loan."
msgstr "Lainausta ei voitu suorittaa."

#: api/problem_details.py:228
msgid "Delivery mechanism conflict."
msgstr "Toimitusmekanismin ristiriita."

#: api/problem_details.py:229
msgid ""
"The delivery mechanism for this book has been locked in and can't be "
"changed."
msgstr "Tämän kirjan toimitusmekanismi on lukittu eikä sitä voi muuttaa."

#: api/problem_details.py:235
msgid "Unsupported delivery mechanism."
msgstr "Toimitusmekanismia ei tueta."

#: api/problem_details.py:236
msgid "You selected a delivery mechanism that's not supported by this book."
msgstr "Valitsit toimitusmekanismin, jota tämä kirja ei tue."

#: api/problem_details.py:242 api/problem_details.py:243
msgid "Could not release hold."
msgstr "Varausta ei voitu vapauttaa."

#: api/problem_details.py:249
msgid "Unknown SAML provider."
msgstr "Tuntematon SAML-palveluntarjoaja."

#: api/problem_details.py:250
msgid "The specified SAML provider name isn't one of the known providers."
msgstr ""
"Määritetty SAML-palveluntarjoajan nimi ei kuulu tunnettuihin "
"palveluntarjoajiin."

#: api/problem_details.py:257
msgid "Ekirjasto provider not configured."
msgstr "E-kirjaston palveluntarjoajaa ei ole määritetty."

#: api/problem_details.py:258
msgid "Ekirjasto provider was not configured for the library"
msgstr "E-kirjaston palveluntarjoajaa ei ole määritetty kirjastolle"

#: api/problem_details.py:264
msgid "Invalid SAML bearer token."
msgstr "Virheellinen SAML:n bearer token."

#: api/problem_details.py:265
msgid "The provided SAML bearer token couldn't be verified."
msgstr "Annettua SAML:n bearer tokenia ei voitu vahvistaa."

#: api/problem_details.py:272
msgid "Invalid delegate token for ekirjasto authentication provider."
msgstr "E-kirjaston todennuksen tarjoajan virheellinen valtuutustunnus."

#: api/problem_details.py:273
msgid ""
"The provided delegate token couldn't be verified for ekirjasto "
"authentication provider or it is expired."
msgstr ""
"E-kirjaston todennuksen tarjoajan antamaa valtuutustunnusta ei voitu "
"vahvistaa tai se on vanhentunut."

#: api/problem_details.py:282
msgid "Invalid ekirjasto token for ekirjasto API."
msgstr "Virheellinen e-kirjaston tunnus e-kirjaston API:lle."

#: api/problem_details.py:283
msgid "The provided ekirjasto token couldn't be verified for ekirjasto API."
msgstr "Annettua e-kirjaston tunnusta ei voitu vahvistaa e-kirjaston API:lle."

#: api/problem_details.py:290
msgid "Authentication with ekirjasto API failed."
msgstr "Todennus e-kirjaston API:lla epäonnistui."

#: api/problem_details.py:291
msgid "Authentication with ekirjasto API failed, for unknown reason."
msgstr "Todennus e-kirjaston API:lla epäonnistui tuntemattomasta syystä."

#: api/problem_details.py:297
msgid "Unsupported authentication mechanism."
msgstr "Todennusmekanismia ei tueta."

#: api/problem_details.py:298
msgid "The specified authentication mechanism isn't supported."
msgstr "Määritettyä todennusmekanismia ei tueta."

#: api/problem_details.py:304
msgid "Invalid analytics event type."
msgstr "Virheellinen analytiikkatapahtuman tyyppi."

#: api/problem_details.py:305
msgid "The analytics event must be a supported type."
msgstr "Analytiikkatapahtuman tyypin on oltava tuettu."

#: api/problem_details.py:311
msgid "Invalid annotation motivation."
msgstr "Virheellinen merkinnän peruste."

#: api/problem_details.py:312
msgid "The annotation must have a supported motivation."
msgstr "Merkinnällä on oltava tuettu peruste."

#: api/problem_details.py:318
msgid "Invalid annotation target."
msgstr "Virheellinen merkinnän kohde."

#: api/problem_details.py:319
msgid "The annotation target must be a work in your current loans."
msgstr "Merkinnän kohteen on oltava nykyisissä lainoissasi oleva teos."

#: api/problem_details.py:325
msgid "Invalid annotation format."
msgstr "Virheellinen merkinnän muoto."

#: api/problem_details.py:326
msgid "The annotation could not be parsed as JSON-LD."
msgstr "Merkintää ei voitu jäsentää JSON-LD-muodossa."

#: api/problem_details.py:332
msgid "No annotation."
msgstr "Ei merkintää."

#: api/problem_details.py:333
msgid "The annotation you requested does not exist."
msgstr "Pyytämääsi merkintää ei ole olemassa."

#: api/problem_details.py:339
msgid "Library not found."
msgstr "Kirjastoa ei löydy."

#: api/problem_details.py:340
msgid "No library with the requested name on this server."
msgstr "Tällä palvelimella ei ole pyydetyn nimistä kirjastoa."

#: api/problem_details.py:346
msgid "Wrong library"
msgstr "Väärä kirjasto"

#: api/problem_details.py:347
msgid "You are not a patron of the selected library."
msgstr "Et ole valitun kirjaston asiakas."

#: api/problem_details.py:353
msgid "Invalid loan for ODL notification"
msgstr "Virheellinen laina ODL-ilmoituksessa"

#: api/problem_details.py:354
msgid "The ODL notification is for a loan that's not from an ODL collection."
msgstr "ODL-ilmoitus koskee lainaa, joka ei ole ODL-kokoelmasta."

#: api/problem_details.py:360
msgid "Invalid registration"
msgstr "Virheellinen rekisteröinti"

#: api/problem_details.py:361
msgid "You did not submit enough information to register with the collection."
msgstr "Et antanut riittävästi tietoja rekisteröityäksesi kokoelmaan."

#: api/problem_details.py:367
msgid "Decryption error"
msgstr "Salauksenpurkuvirhe"

#: api/problem_details.py:368
msgid "Failed to decrypt a shared secret retrieved from another computer."
msgstr ""
"Toisesta tietokoneesta haetun jaetun salauksen purkaminen epäonnistui."

#: api/problem_details.py:374
msgid "Device token not found"
msgstr "Laitetunnusta ei löydy"

#: api/problem_details.py:375
msgid "Patron does not have a device registered with this token."
msgstr "Asiakkaalla ei ole tällä tunnuksella rekisteröityä laitetta."

#: api/problem_details.py:381
msgid "Device token already exists"
msgstr "Laitetunnus on jo olemassa"

#: api/problem_details.py:382
msgid "A device token with the same token already exists."
msgstr "Laitetunnus samalla tunnuksella on jo olemassa."

#: api/problem_details.py:388
msgid "Device token type invalid"
msgstr "Virheellinen laitetunnustyyppi"

#: api/problem_details.py:389
msgid "The token type provided is not valid."
msgstr "Annettu tunnustyyppi ei kelpaa."

#: api/problem_details.py:395
msgid "Access token expired"
msgstr "Käyttötunnus vanhentunut"

#: api/problem_details.py:396
msgid "The patron authentication access token has expired."
msgstr "Asiakkaan todennuksen käyttötunnus on vanhentunut."

#: api/problem_details.py:402
msgid "Access token invalid"
msgstr "Virheellinen käyttötunnus"

#: api/problem_details.py:403
msgid "The patron authentication access token is invalid."
msgstr "Asiakkaan todennuksen käyttötunnus on virheellinen."

#: api/problem_details.py:409
msgid "Access token not possible"
msgstr "Käyttötunnus ei ole mahdollinen"

#: api/problem_details.py:410
msgid ""
"The patron authentication access token is not possible for this type of "
"authentication."
msgstr ""
"Asiakkaan todennuksen käyttötunnus ei ole mahdollinen tämäntyyppisessä "
"todennuksessa."

#: api/sirsidynix_authentication_provider.py:32
msgid "Patron has not yet been approved"
msgstr "Asiakasta ei ole vielä hyväksytty"

#: api/sirsidynix_authentication_provider.py:33
msgid "Patron membership has expired"
msgstr "Asiakkaan jäsenyys on päättynyt"

#: api/sirsidynix_authentication_provider.py:34
msgid "Patron has been blocked."
msgstr "Asiakas on estetty."
# English translations for PROJECT.
# Copyright (C) 2024 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2024.
# 
# Translators:
# Marika Kaskinen, 2024
# 
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2024-02-23 15:11+0200\n"
"PO-Revision-Date: 2024-03-01 11:37+0000\n"
"Last-Translator: Marika Kaskinen, 2024\n"
"Language-Team: Finnish (https://app.transifex.com/national-library-of-finland/teams/183401/fi/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.13.0\n"
"Language: fi\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: core/config.py:95
msgid "Debug"
msgstr "Virheenkorjaus"

#: core/config.py:96
msgid "Info"
msgstr "Tiedot"

#: core/config.py:97
msgid "Warn"
msgstr "Varoitus"

#: core/config.py:98
msgid "Error"
msgstr "Virhe"

#: core/config.py:106
msgid "Base url of the application"
msgstr "Sovelluksen perus-url"

#: core/config.py:112
msgid "Log Level"
msgstr "Lokitaso"

#: core/config.py:119
msgid "Application name"
msgstr "Sovelluksen nimi"

#: core/config.py:120
msgid ""
"Log messages originating from this application will be tagged with this "
"name. If you run multiple instances, giving each one a different application"
" name will help you determine which instance is having problems."
msgstr ""
"Tästä sovelluksesta peräisin olevat lokiviestit merkitään tällä nimellä. Jos"
" suoritat useita esiintymiä, voit antaa kullekin eri sovelluksen nimen, "
"jotta sinun on helpompi määrittää, missä esiintymässä on ongelmia."

#: core/config.py:128
msgid "Database Log Level"
msgstr "Tietokantalokitaso"

#: core/config.py:131
msgid ""
"Database logs are extremely verbose, so unless you're diagnosing a database-"
"related problem, it's a good idea to set a higher log level for database "
"messages."
msgstr ""
"Tietokantalokit ovat erittäin monisanaisia, joten ellei kyseessä ole "
"tietokantaan liittyvän ongelman määrittäminen, tietokantaviesteille "
"kannattaa asettaa korkeampi lokitaso."

#: core/config.py:138
msgid "Excluded audiobook sources"
msgstr "Poissuljetut äänikirjalähteet"

#: core/config.py:139
msgid ""
"Audiobooks from these data sources will be hidden from the collection, even "
"if they would otherwise show up as available."
msgstr ""
"Näistä tietolähteistä peräisin olevat äänikirjat piilotetaan kokoelmasta, "
"vaikka ne muuten näkyisivät saatavilla olevina."

#: core/config.py:147
msgid "Cleanup old measurement data"
msgstr "Vanhojen mittaustietojen puhdistus"

#: core/config.py:149
msgid ""
"If this settings is 'true' old book measurement data will be cleaned out of "
"the database. Some sites may want to keep this data for later analysis."
msgstr ""
"Jos tämän asetuksen arvona on \"tosi\", kirjan vanhat mittaustiedot "
"poistetaan tietokannasta. Jotkut sivustot saattavat haluta säilyttää nämä "
"tiedot myöhempää analysointia varten."

#: core/config.py:157
msgid "Push notifications status"
msgstr "Push-ilmoitusten tila"

#: core/config.py:159
msgid ""
"If this settings is 'true' push notification jobs will run as scheduled, and"
" attempt to notify patrons via mobile push notifications."
msgstr ""
"Jos tämän asetuksen arvona on \"tosi\", push-ilmoitustehtävät suoritetaan "
"aikataulun mukaisesti, ja asiakkaille yritetään ilmoittaa mobiililaitteille "
"tulevien push-ilmoitusten kautta."

#: core/config.py:163
msgid "True"
msgstr "Tosi"

#: core/config.py:164
msgid "False"
msgstr "Epätosi"

#: core/connection_config.py:17
msgid "Connection retry limit"
msgstr "Yhteyden uudelleen yrittämisen raja"

#: core/connection_config.py:18
msgid ""
"The maximum number of times to retry a request for certain connection-"
"related errors."
msgstr ""
"Kuinka monta kertaa pyyntöä voi enintään yrittää uudelleen tietyissä "
"yhteyteen liittyvissä virheissä."

#: core/entrypoint.py:103 core/facets.py:97
msgid "All"
msgstr "Kaikki"

#: core/entrypoint.py:140
msgid "eBooks"
msgstr "E-kirjat"

#: core/entrypoint.py:149
msgid "Audiobooks"
msgstr "Äänikirjat"

#: core/external_search.py:108 core/opds_import.py:110
msgid "URL"
msgstr "URL-osoite"

#: core/external_search.py:114
msgid "Index prefix"
msgstr "Hakemiston etuliite"

#: core/external_search.py:117
msgid ""
"Any Search indexes needed for this application will be created with this "
"unique prefix. In most cases, the default will work fine. You may need to "
"change this if you have multiple application servers using a single Search "
"server."
msgstr ""
"Kaikki tämän sovelluksen tarvitsemat hakuindeksit luodaan tällä "
"yksilöllisellä etuliitteellä. Useimmissa tapauksissa oletusarvo toimii "
"hyvin. Tätä on ehkä muutettava, jos useat sovelluspalvelimet käyttävät yhtä "
"hakupalvelinta."

#: core/external_search.py:123
msgid "Test search term"
msgstr "Testin hakusana"

#: core/external_search.py:125
msgid "Self tests will use this value as the search term."
msgstr "Itsetestit käyttävät tätä arvoa hakusanana."

#: core/external_search.py:2628
#, python-format
msgid "Invalid page key: %(key)s"
msgstr "Virheellinen sivuavain: %(key)s"

#: core/facets.py:72
msgid "Sort by"
msgstr "Lajitteluperuste"

#: core/facets.py:73
msgid "Availability"
msgstr "Saatavuus"

#: core/facets.py:74
msgid "Collection"
msgstr "Kokoelma"

#: core/facets.py:75
msgid "Distributor"
msgstr "Jakelija"

#: core/facets.py:76
msgid "Collection Name"
msgstr "Kokoelman nimi"

#: core/facets.py:80
msgid "Allow patrons to sort by"
msgstr "Salli asiakkaiden lajitella seuraavan mukaan"

#: core/facets.py:81
msgid "Allow patrons to filter availability to"
msgstr "Salli asiakkaiden suodattaa saatavuus"

#: core/facets.py:82
msgid "Allow patrons to filter collection to"
msgstr "Salli asiakkaiden suodattaa kokoelma"

#: core/facets.py:83
msgid "Allow patrons to filter by distributor"
msgstr "Salli asiakkaiden suodattaa jakelijan mukaan"

#: core/facets.py:84
msgid "Allow patrons to filter by collection name"
msgstr "Salli asiakkaiden suodattaa kokoelman nimen mukaan"

#: core/facets.py:90
msgid "Title"
msgstr "Nimi"

#: core/facets.py:91
msgid "Author"
msgstr "Tekijä"

#: core/facets.py:92
msgid "Last Update"
msgstr "Viimeisin päivitys"

#: core/facets.py:93
msgid "Recently Added"
msgstr "Äskettäin lisätty"

#: core/facets.py:94
msgid "Series Position"
msgstr "Sarjan sijainti"

#: core/facets.py:95
msgid "Work ID"
msgstr "Työn tunnus"

#: core/facets.py:96
msgid "Available now"
msgstr "Saatavilla nyt"

#: core/facets.py:98
msgid "Yours to keep"
msgstr "Voit pitää tämän"

#: core/facets.py:99
msgid "Everything"
msgstr "Kaikki"

#: core/facets.py:100
msgid "Popular Books"
msgstr "Suositut kirjat"

#: core/lane.py:380
#, python-format
msgid "I don't know how to order a feed by '%(order)s'"
msgstr "En tiedä, miten syöte järjestetään seuraavan mukaan: %(order)s"

#: core/lane.py:388
#, python-format
msgid "I don't understand the availability term '%(availability)s'"
msgstr "En ymmärrä saatavuustermiä %(availability)s"

#: core/lane.py:400
#, python-format
msgid "I don't understand what '%(collection)s' refers to."
msgstr "En ymmärrä, mitä %(collection)s tarkoittaa."

#: core/lane.py:416
#, python-format
msgid "I don't understand which distributor '%(distributor)s' refers to."
msgstr "En ymmärrä, mihin jakelijaan %(distributor)s viittaa."

#: core/lane.py:432
#, python-format
msgid "I don't understand which collection '%(collection_name)s' refers to."
msgstr "En ymmärrä, mihin kokoelmaan %(collection_name)s viittaa."

#: core/lane.py:1216
#, python-format
msgid "Invalid page size: %(size)s"
msgstr "Virheellinen sivukoko: %(size)s"

#: core/lane.py:1234
#, python-format
msgid "Invalid offset: %(offset)s"
msgstr "Virheellinen offset: %(offset)s"

#: core/lane.py:1555
#, python-format
msgid "All %(worklist)s"
msgstr "Kaikki %(worklist)s"

#: core/opds2_import.py:194 core/opds_import.py:166
msgid "Custom accept header"
msgstr "Mukautettu hyväksymisen otsikko"

#: core/opds2_import.py:195 core/opds_import.py:168
msgid ""
"Some servers expect an accept header to decide which file to send. You can "
"use */* if the server doesn't expect anything."
msgstr ""
"Jotkin palvelimet olettavat, että hyväksymisen otsikko päättää, mikä "
"tiedosto lähetetään. Voit käyttää merkkejä */*, jos palvelin ei oleta "
"mitään."

#: core/opds2_import.py:207
msgid "List of identifiers that will be skipped"
msgstr "Luettelo ohitettavista tunnisteista"

#: core/opds2_import.py:208
msgid ""
"Circulation Manager will not be importing publications with identifiers "
"having one of the selected types."
msgstr ""
"Lainauksen hallinta ei tuo julkaisuja, joiden tunnisteilla on jokin "
"valituista tyypeistä."

#: core/opds2_import.py:326
msgid "Import books from a publicly-accessible OPDS 2.0 feed."
msgstr "Tuo kirjoja julkisesti saatavilla olevasta OPDS 2.0 -syötteestä."

#: core/opds_import.py:116
msgid "Data source name"
msgstr "Tietolähteen nimi"

#: core/opds_import.py:122
msgid "Default audience"
msgstr "Oletusyleisö"

#: core/opds_import.py:123
msgid ""
"If the vendor does not specify the target audience for their books, assume "
"the books have this target audience."
msgstr ""
"Jos toimittaja ei määritä kirjojensa kohdeyleisöä, oleta, että kirjojen "
"kohdeyleisö on tämä."

#: core/opds_import.py:129
msgid "No default audience"
msgstr "Ei oletusyleisöä"

#: core/opds_import.py:138
msgid "Username"
msgstr "Käyttäjänimi"

#: core/opds_import.py:139
msgid ""
"If HTTP Basic authentication is required to access the OPDS feed (it usually"
" isn't), enter the username here."
msgstr ""
"Jos HTTP Basic -tunnistus vaaditaan OPDS-syötteen käyttämiseksi (yleensä "
"sitä ei vaadita), kirjoita käyttäjänimi tähän."

#: core/opds_import.py:148
msgid "Password"
msgstr "Salasana"

#: core/opds_import.py:149
msgid ""
"If HTTP Basic authentication is required to access the OPDS feed (it usually"
" isn't), enter the password here."
msgstr ""
"Jos HTTP Basic -tunnistus vaaditaan OPDS-syötteen käyttämiseksi (yleensä "
"sitä ei vaadita), kirjoita salasana tähän."

#: core/opds_import.py:177
msgid "Identifer"
msgstr "Tunniste"

#: core/opds_import.py:179
msgid "Which book identifier to use as ID."
msgstr "Mitä kirjan tunnistetta käytetään tunnuksena."

#: core/opds_import.py:182
msgid "(Default) Use <id>"
msgstr "(Oletus) Käytä <id>"

#: core/opds_import.py:183
msgid "Use <dcterms:identifier> first, if not exist use <id>"
msgstr "Käytä ensin <dcterms:identifier>, jos sitä ei ole, käytä <id>"

#: core/opds_import.py:654
msgid "Import books from a publicly-accessible OPDS feed."
msgstr "Tuo kirjoja julkisesti saatavilla olevasta OPDS-syötteestä."

#: core/problem_details.py:11
msgid "Invalid input."
msgstr "Virheellinen syöte."

#: core/problem_details.py:12
msgid "You provided invalid or unrecognized input."
msgstr "Annoit virheellisen tai tunnistamattoman syötteen."

#: core/problem_details.py:18
msgid "Invalid credentials"
msgstr "Virheelliset kirjautumistiedot"

#: core/problem_details.py:19
msgid "Valid credentials are required."
msgstr "Kelvolliset kirjautumistiedot vaaditaan."

#: core/problem_details.py:25
msgid "Method not allowed"
msgstr "Menetelmä ei ole sallittu"

#: core/problem_details.py:26
msgid "The HTTP method you used is not allowed on this resource."
msgstr "Käyttämäsi HTTP-menetelmä ei ole sallittu tässä resurssissa."

#: core/problem_details.py:32
msgid "Unsupported media type"
msgstr "Mediatyyppi, jota ei tueta"

#: core/problem_details.py:33
msgid "You submitted an unsupported media type."
msgstr "Annoit mediatyypin, jota ei tueta."

#: core/problem_details.py:39
msgid "Payload too large"
msgstr "Liian suuri kuormitus"

#: core/problem_details.py:40
msgid "You submitted a document that was too large."
msgstr "Annoit liian suuren dokumentin."

#: core/problem_details.py:46
msgid "Internal server error."
msgstr "Sisäinen palvelinvirhe."

#: core/problem_details.py:47
msgid "Internal server error"
msgstr "Sisäinen palvelinvirhe"

#: core/problem_details.py:56
msgid "Invalid URN"
msgstr "Virheellinen URN"

#: core/problem_details.py:57
msgid "Could not parse identifier."
msgstr "Tunnistetta ei voitu jäsentää."

#: core/problem_details.py:63
msgid "Unrecognized data source."
msgstr "Tunnistamaton tietolähde."

#: core/problem_details.py:64
msgid "I don't know anything about that data source."
msgstr "En tiedä mitään kyseisestä tietolähteestä."

#: core/problem_details.py:70
msgid "Third-party service failed."
msgstr "Kolmannen osapuolen palvelu epäonnistui."

#: core/problem_details.py:71
msgid "A third-party service has failed."
msgstr "Kolmannen osapuolen palvelu on epäonnistunut."

#: core/problem_details.py:77
msgid "Invalid reset password token"
msgstr "Virheellinen salasanan nollauksen tunnus"

#: core/problem_details.py:78
msgid "Provided reset password token is not valid."
msgstr "Annettu salasanan nollauksen tunnus ei kelpaa."

#: core/user_profile.py:41
#, python-format
msgid "Profile document is not a JSON object: %r."
msgstr "Profiilidokumentti ei ole JSON-objekti: %r."

#: core/user_profile.py:47
#, python-format
msgid "Could not convert profile document to JSON: %r."
msgstr "Profiilidokumenttia ei voitu muuttaa JSON:ksi: %r."

#: core/user_profile.py:62
#, python-format
msgid "Expected %s"
msgstr "Odotettu %s"

#: core/user_profile.py:67
msgid "Submitted profile document was not valid JSON."
msgstr "Annettu profiilidokumentti ei ollut kelvollinen JSON."

#: core/user_profile.py:71
msgid "Submitted profile document was not a JSON object."
msgstr "Annettu profiilidokumentti ei ollut JSON-objekti."

#: core/user_profile.py:83
#, python-format
msgid "\"%s\" is not a writable setting."
msgstr "%s ei ole kirjoitettava asetus."

#: core/classifier/localized_names.py:4
msgid "Adventure"
msgstr "Seikkailu"

#: core/classifier/localized_names.py:5
msgid "Classics"
msgstr "Klassikot"

#: core/classifier/localized_names.py:6
msgid "Comics & Graphic Novels"
msgstr "Sarjakuvalehdet ja -romaanit"

#: core/classifier/localized_names.py:7
msgid "Drama"
msgstr "Draama"

#: core/classifier/localized_names.py:8
msgid "Erotica"
msgstr "Erotiikka"

#: core/classifier/localized_names.py:9
msgid "Fantasy"
msgstr "Fantasia"

#: core/classifier/localized_names.py:10
msgid "Epic Fantasy"
msgstr "Eeppinen fantasia"

#: core/classifier/localized_names.py:11
msgid "Historical Fantasy"
msgstr "Historiallinen fantasia"

#: core/classifier/localized_names.py:12
msgid "Urban Fantasy"
msgstr "Urbaani fantasia"

#: core/classifier/localized_names.py:13
msgid "Folklore"
msgstr "Kansanperinne"

#: core/classifier/localized_names.py:14
msgid "Historical Fiction"
msgstr "Historiallinen fiktio"

#: core/classifier/localized_names.py:15
msgid "Horror"
msgstr "Kauhu"

#: core/classifier/localized_names.py:16
msgid "Gothic Horror"
msgstr "Goottilainen kauhu"

#: core/classifier/localized_names.py:17
msgid "Ghost Stories"
msgstr "Kummitustarinat"

#: core/classifier/localized_names.py:18
msgid "Vampires"
msgstr "Vampyyrit"

#: core/classifier/localized_names.py:19
msgid "Werewolves"
msgstr "Ihmissudet"

#: core/classifier/localized_names.py:20
msgid "Occult Horror"
msgstr "Yliluonnollinen kauhu"

#: core/classifier/localized_names.py:21
msgid "Humorous Fiction"
msgstr "Humoristinen fiktio"

#: core/classifier/localized_names.py:22
msgid "Literary Fiction"
msgstr "Kaunokirjallisuus"

#: core/classifier/localized_names.py:23
msgid "LGBTQ Fiction"
msgstr "HLBTQ-fiktio"

#: core/classifier/localized_names.py:24
msgid "Mystery"
msgstr "Dekkarit"

#: core/classifier/localized_names.py:25
msgid "Crime & Detective Stories"
msgstr "Rikos- ja salapoliisiromaanit"

#: core/classifier/localized_names.py:26
msgid "Hard-Boiled Mystery"
msgstr "Kovaksikeitetyt dekkarit"

#: core/classifier/localized_names.py:27
msgid "Police Procedural"
msgstr "Poliisiromaanit"

#: core/classifier/localized_names.py:28
msgid "Cozy Mystery"
msgstr "Hyvän mielen dekkarit"

#: core/classifier/localized_names.py:29
msgid "Historical Mystery"
msgstr "Historialliset dekkarit"

#: core/classifier/localized_names.py:30
msgid "Paranormal Mystery"
msgstr "Yliluonnolliset dekkarit"

#: core/classifier/localized_names.py:31
msgid "Women Detectives"
msgstr "Naisetsivät"

#: core/classifier/localized_names.py:32
msgid "Poetry"
msgstr "Runous"

#: core/classifier/localized_names.py:33
msgid "Religious Fiction"
msgstr "Uskonnollinen fiktio"

#: core/classifier/localized_names.py:34
msgid "Romance"
msgstr "Romantiikka"

#: core/classifier/localized_names.py:35
msgid "Contemporary Romance"
msgstr "Nykyajan romantiikka"

#: core/classifier/localized_names.py:36
msgid "Gothic Romance"
msgstr "Goottilainen romantiikka"

#: core/classifier/localized_names.py:37
msgid "Historical Romance"
msgstr "Historiallinen romantiikka"

#: core/classifier/localized_names.py:38
msgid "Paranormal Romance"
msgstr "Yliluonnollinen romantiikka"

#: core/classifier/localized_names.py:39
msgid "Western Romance"
msgstr "Lännen romantiikka"

#: core/classifier/localized_names.py:40
msgid "Romantic Suspense"
msgstr "Romanttinen jännitys"

#: core/classifier/localized_names.py:41
msgid "Science Fiction"
msgstr "Science Fiction"

#: core/classifier/localized_names.py:42
msgid "Dystopian SF"
msgstr "Dystooppinen scifi"

#: core/classifier/localized_names.py:43
msgid "Space Opera"
msgstr "Avaruusooppera"

#: core/classifier/localized_names.py:44
msgid "Cyberpunk"
msgstr "Kyberpunk"

#: core/classifier/localized_names.py:45
msgid "Military SF"
msgstr "Sotilaallinen scifi"

#: core/classifier/localized_names.py:46
msgid "Alternative History"
msgstr "Vaihtoehtoinen historia"

#: core/classifier/localized_names.py:47
msgid "Steampunk"
msgstr "Steampunk"

#: core/classifier/localized_names.py:48
msgid "Romantic SF"
msgstr "Romanttinen scifi"

#: core/classifier/localized_names.py:49
msgid "Media Tie-in SF"
msgstr "TV:stä tuttu scifi"

#: core/classifier/localized_names.py:50
msgid "Short Stories"
msgstr "Novellit"

#: core/classifier/localized_names.py:51
msgid "Suspense/Thriller"
msgstr "Jännitys/trilleri"

#: core/classifier/localized_names.py:52
msgid "Historical Thriller"
msgstr "Historialliset trillerit"

#: core/classifier/localized_names.py:53
msgid "Espionage"
msgstr "Vakoilu"

#: core/classifier/localized_names.py:54
msgid "Supernatural Thriller"
msgstr "Yliluonnolliset trillerit"

#: core/classifier/localized_names.py:55
msgid "Medical Thriller"
msgstr "Lääketieteelliset trillerit"

#: core/classifier/localized_names.py:56
msgid "Political Thriller"
msgstr "Poliittiset trillerit"

#: core/classifier/localized_names.py:57
msgid "Psychological Thriller"
msgstr "Psykologiset trillerit"

#: core/classifier/localized_names.py:58
msgid "Technothriller"
msgstr "Teknotrillerit"

#: core/classifier/localized_names.py:59
msgid "Legal Thriller"
msgstr "Oikeudenkäyntitrillerit"

#: core/classifier/localized_names.py:60
msgid "Military Thriller"
msgstr "Sotilaalliset trillerit"

#: core/classifier/localized_names.py:61
msgid "Urban Fiction"
msgstr "Urbaani fiktio"

#: core/classifier/localized_names.py:62
msgid "Westerns"
msgstr "Westernit"

#: core/classifier/localized_names.py:63
msgid "Women's Fiction"
msgstr "Naisten fiktio"

#: core/classifier/localized_names.py:64
msgid "Art & Design"
msgstr "Taide ja muotoilu"

#: core/classifier/localized_names.py:65
msgid "Architecture"
msgstr "Arkkitehtuuri"

#: core/classifier/localized_names.py:66
msgid "Art"
msgstr "Taide"

#: core/classifier/localized_names.py:67
msgid "Art Criticism & Theory"
msgstr "Taidekritiikki ja taiteen teoria"

#: core/classifier/localized_names.py:68
msgid "Art History"
msgstr "Taidehistoria"

#: core/classifier/localized_names.py:69
msgid "Design"
msgstr "Muotoilu"

#: core/classifier/localized_names.py:70
msgid "Fashion"
msgstr "Muoti"

#: core/classifier/localized_names.py:71
msgid "Photography"
msgstr "Valokuvaus"

#: core/classifier/localized_names.py:72
msgid "Biography & Memoir"
msgstr "Elämäkerrat ja muistelmat"

#: core/classifier/localized_names.py:73
msgid "Education"
msgstr "Koulutus"

#: core/classifier/localized_names.py:74
msgid "Personal Finance & Business"
msgstr "Henkilökohtainen talous ja liiketoiminta"

#: core/classifier/localized_names.py:75
msgid "Business"
msgstr "Liiketoiminta"

#: core/classifier/localized_names.py:76
msgid "Economics"
msgstr "Taloustiede"

#: core/classifier/localized_names.py:77
msgid "Management & Leadership"
msgstr "Johtaminen ja johtajuus"

#: core/classifier/localized_names.py:78
msgid "Personal Finance & Investing"
msgstr "Henkilökohtainen talous ja sijoittaminen"

#: core/classifier/localized_names.py:79
msgid "Real Estate"
msgstr "Kiinteistöt"

#: core/classifier/localized_names.py:80
msgid "Parenting & Family"
msgstr "Vanhemmuus ja perhe"

#: core/classifier/localized_names.py:81
msgid "Family & Relationships"
msgstr "Perhe ja ihmissuhteet"

#: core/classifier/localized_names.py:82
msgid "Parenting"
msgstr "Vanhemmuus"

#: core/classifier/localized_names.py:83
msgid "Food & Health"
msgstr "Ruoka ja terveys"

#: core/classifier/localized_names.py:84
msgid "Bartending & Cocktails"
msgstr "Baarimestari ja cocktailit"

#: core/classifier/localized_names.py:85
msgid "Cooking"
msgstr "Ruoanlaitto"

#: core/classifier/localized_names.py:86
msgid "Health & Diet"
msgstr "Terveys ja ruokavaliot"

#: core/classifier/localized_names.py:87
msgid "Vegetarian & Vegan"
msgstr "Kasvissyönti ja veganismi"

#: core/classifier/localized_names.py:88
msgid "History"
msgstr "Historia"

#: core/classifier/localized_names.py:89
msgid "African History"
msgstr "Afrikan historia"

#: core/classifier/localized_names.py:90
msgid "Ancient History"
msgstr "Muinaishistoria"

#: core/classifier/localized_names.py:91
msgid "Asian History"
msgstr "Aasian historia"

#: core/classifier/localized_names.py:92
msgid "Civil War History"
msgstr "Sisällissodan historia"

#: core/classifier/localized_names.py:93
msgid "European History"
msgstr "Euroopan historia"

#: core/classifier/localized_names.py:94
msgid "Latin American History"
msgstr "Latinalaisen Amerikan historia"

#: core/classifier/localized_names.py:95
msgid "Medieval History"
msgstr "Keskiajan historia"

#: core/classifier/localized_names.py:96
msgid "Middle East History"
msgstr "Lähi-idän historia"

#: core/classifier/localized_names.py:97
msgid "Military History"
msgstr "Sotahistoria"

#: core/classifier/localized_names.py:98
msgid "Modern History"
msgstr "Uuden ajan historia"

#: core/classifier/localized_names.py:99
msgid "Renaissance & Early Modern History"
msgstr "Renessanssi ja uuden ajan alun historia"

#: core/classifier/localized_names.py:100
msgid "United States History"
msgstr "Yhdysvaltojen historia"

#: core/classifier/localized_names.py:101
msgid "World History"
msgstr "Maailmanhistoria"

#: core/classifier/localized_names.py:102
msgid "Hobbies & Home"
msgstr "Harrastukset ja koti"

#: core/classifier/localized_names.py:103
msgid "Antiques & Collectibles"
msgstr "Antiikki ja keräilyesineet"

#: core/classifier/localized_names.py:104
msgid "Crafts & Hobbies"
msgstr "Käsityöt ja harrastukset"

#: core/classifier/localized_names.py:105
msgid "Gardening"
msgstr "Puutarhanhoito"

#: core/classifier/localized_names.py:106
msgid "Games"
msgstr "Pelit"

#: core/classifier/localized_names.py:107
msgid "House & Home"
msgstr "Talo ja koti"

#: core/classifier/localized_names.py:108
msgid "Pets"
msgstr "Lemmikkieläimet"

#: core/classifier/localized_names.py:109
msgid "Humorous Nonfiction"
msgstr "Humoristinen tietokirjallisuus"

#: core/classifier/localized_names.py:110
msgid "Entertainment"
msgstr "Viihde"

#: core/classifier/localized_names.py:111
msgid "Film & TV"
msgstr "Elokuvat ja televisio"

#: core/classifier/localized_names.py:112
msgid "Music"
msgstr "Musiikki"

#: core/classifier/localized_names.py:113
msgid "Performing Arts"
msgstr "Esittävät taiteet"

#: core/classifier/localized_names.py:114
msgid "Life Strategies"
msgstr "Elämänstrategiat"

#: core/classifier/localized_names.py:115
msgid "Literary Criticism"
msgstr "Kirjallisuuskritiikki"

#: core/classifier/localized_names.py:116
msgid "Periodicals"
msgstr "Aikakauslehdet"

#: core/classifier/localized_names.py:117
msgid "Philosophy"
msgstr "Filosofia"

#: core/classifier/localized_names.py:118
msgid "Political Science"
msgstr "Valtiotiede"

#: core/classifier/localized_names.py:119
msgid "Reference & Study Aids"
msgstr "Tietokirjat ja opintoapu"

#: core/classifier/localized_names.py:120
msgid "Dictionaries"
msgstr "Sanakirjat"

#: core/classifier/localized_names.py:121
msgid "Foreign Language Study"
msgstr "Vieraiden kielten opiskelu"

#: core/classifier/localized_names.py:122
msgid "Law"
msgstr "Laki"

#: core/classifier/localized_names.py:123
msgid "Study Aids"
msgstr "Opintoapu"

#: core/classifier/localized_names.py:124
msgid "Religion & Spirituality"
msgstr "Uskonto ja hengellisyys"

#: core/classifier/localized_names.py:125
msgid "Body, Mind & Spirit"
msgstr "Keho, mieli ja henki"

#: core/classifier/localized_names.py:126
msgid "Buddhism"
msgstr "Buddhalaisuus"

#: core/classifier/localized_names.py:127
msgid "Christianity"
msgstr "Kristinusko"

#: core/classifier/localized_names.py:128
msgid "Hinduism"
msgstr "Hindulaisuus"

#: core/classifier/localized_names.py:129
msgid "Islam"
msgstr "Islam"

#: core/classifier/localized_names.py:130
msgid "Judaism"
msgstr "Juutalaisuus"

#: core/classifier/localized_names.py:131
msgid "Science & Technology"
msgstr "Tiede ja teknologia"

#: core/classifier/localized_names.py:132
msgid "Computers"
msgstr "Tietokoneet"

#: core/classifier/localized_names.py:133
msgid "Mathematics"
msgstr "Matematiikka"

#: core/classifier/localized_names.py:134
msgid "Medical"
msgstr "Lääketiede"

#: core/classifier/localized_names.py:135
msgid "Nature"
msgstr "Luonto"

#: core/classifier/localized_names.py:136
msgid "Psychology"
msgstr "Psykologia"

#: core/classifier/localized_names.py:137
msgid "Science"
msgstr "Tiede"

#: core/classifier/localized_names.py:138
msgid "Social Sciences"
msgstr "Yhteiskuntatieteet"

#: core/classifier/localized_names.py:139
msgid "Technology"
msgstr "Teknologia"

#: core/classifier/localized_names.py:140
msgid "Self-Help"
msgstr "Oma-apu"

#: core/classifier/localized_names.py:141
msgid "Sports"
msgstr "Urheilu"

#: core/classifier/localized_names.py:142
msgid "Travel"
msgstr "Matkailu"

#: core/classifier/localized_names.py:143
msgid "True Crime"
msgstr "True Crime"

#: core/util/http.py:27
msgid "Failure contacting external service"
msgstr "Yhteydenotto ulkoiseen palveluun epäonnistui"

#: core/util/http.py:28
#, python-format
msgid ""
"The server tried to access %(service)s but the third-party service "
"experienced an error."
msgstr ""
"Palvelin yritti käyttää palvelua %(service)s, mutta kolmannen osapuolen "
"palvelussa tapahtui virhe."

#: core/util/http.py:75
msgid "Bad response"
msgstr "Väärä vastaus"

#: core/util/http.py:76
#, python-format
msgid ""
"The server made a request to %(service)s, and got an unexpected or invalid "
"response."
msgstr ""
"Palvelin teki pyynnön palveluun %(service)s ja sai odottamattoman tai "
"virheellisen vastauksen."

#: core/util/http.py:148
msgid "Network failure contacting third-party service"
msgstr "Verkkovirhe yhteydenotossa kolmannen osapuolen palveluun"

#: core/util/http.py:149
#, python-format
msgid "The server experienced a network error while contacting %(service)s."
msgstr ""
"Palvelimella tapahtui verkkovirhe, kun se otti yhteyttä palveluun "
"%(service)s."

#: core/util/http.py:158
msgid "Timeout"
msgstr "Aikakatkaisu"

#: core/util/http.py:159
#, python-format
msgid "The server made a request to %(service)s, and that request timed out."
msgstr ""
"Palvelin teki pyynnön palveluun %(service)s, ja pyyntö aikakatkaistiin."
